package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@s-ui/tealium-tracker",
+  "version": "1.0.0",
+  "description": "Tool to track tealium tags and handle events with tealium",
+  "main": "lib/",
+  "scripts": {
+    "lib": "npm run lib:js",
+    "lib:js": "rm -Rf ./lib && mkdir -p ./lib && babel --presets sui ./src --out-dir ./lib",
```

Comment:
I really think that we need a bundler for libs

Reply:
Agree!! We can talk about that what do you have in mind?